internal/vcs/git/main_test.go

```diff
@@ -38,7 +39,7 @@ func TestMain(m *testing.M) {
 	flag.Parse()
 
 	if !testing.Verbose() {
-		log15.Root().SetHandler(log15.DiscardHandler())
+		logtest.Init(m)
```

Comment:
```suggestion
		logtest.InitWithLevel(m, sglog.LevelNone)
```

is the correct equivalent of `DiscardHandler`